lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/AbstractPostgresKgStore.scala

```diff
@@ -18,16 +19,34 @@ abstract class AbstractPostgresKgStore(protected val databaseConfigProvider: Pos
   protected lazy val edges = TableQuery[EdgeTable]
   protected lazy val edgeLabels = TableQuery[EdgeLabelTable]
   protected lazy val edgeSources = TableQuery[EdgeSourceTable]
-  protected lazy val nodes = TableQuery[NodeTable]
-  protected lazy val nodeLabels = TableQuery[NodeLabelTable]
+  protected object nodes extends TableQuery(new NodeTable(_)) {
+    def withNodeLabels = for {
```

Comment:
I think you probably need to pass the node id / node label you want into this. I'd have to see the SQL that results from using this, though, to see if it's pulling the whole table and then selecting what you want, or doing one operation.